04/app.js

```diff
@@ -1 +1,20 @@
-const years = [1980, 1934, 2002, 2019];
\ No newline at end of file
+const years = [1980, 1934, 2002, 2019];
+let yearsDifference;
+let yearsDifferences = [];
```

Comment:
Uwaga na nazwy zmiennych. Różnica tylko 1 znakiem tj. `s` może powodować prostą pomyłkę. 
Warto mieć to na uwadze przy toworzeniu nazw zmiennych ;)

Reply:
ok